### ISSUE TITLE
numpy error when running go_web.bat

### ISSUE THREAD

**Issue by FlashlightET** [RVC-Project/Retrieval-based-Voice-Conversion-WebUI#464]
```File "__init__.pxd", line 942, in numpy.import_array
RuntimeError: module compiled against API version 0x10 but this version of numpy is 0xf

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "H:\blah blah had to move these\X\RVC\RVC-beta-v2-0528\infer-web.py", line 84, in <module>
    from vc_infer_pipeline import VC
  File "H:\blah blah had to move these\X\RVC\RVC-beta-v2-0528\vc_infer_pipeline.py", line 5, in <module>
    import pyworld, os, traceback, faiss, librosa,torchcrepe
  File "H:\blah blah had to move these\X\RVC\RVC-beta-v2-0528\runtime\lib\site-packages\pyworld\__init__.py", line 7, in <module>
    from .pyworld import *
  File "pyworld\pyworld.pyx", line 6, in init pyworld.pyworld
  File "__init__.pxd", line 944, in numpy.import_array
ImportError: numpy.core.multiarray failed to import```

**Comment by RVC-Boss** [comment]
Is there another python runtime in your system?

**Comment by Rajendayo** [comment]
I faced the same issue and it turns out there IS another python runtime, I went to "C:\Users\user\AppData\Roaming\Python" and removed Python39 and it works. It was an installation from an old project so yeah.

**Comment by FlashlightET** [comment]
all i had to do to fix it was remove numpy from a python 3.9 i also had on
my system that i dont know where i used it
